### ISSUE TITLE
Clear using deleteAll and updateAll

### ISSUE THREAD

**Issue by elgrones** [jeremyharris/cacher#15]
Hi
cache cleaning does not work when using the methods
deleteAll() and updateAll()

**Comment by jeremyharris** [comment]
Yes, this is because `deleteAll()` and `updateAll()` do not trigger callbacks by default, which are what Cacher uses to determine what file it should clean. While those methods support triggering callbacks (third param), Cacher doesn't officially support them and there are no tests for them. (From what I remember it probably wouldn't work by default because the conditions [which are used to determine what cache file to remove] used in the `deleteAll` call likely don't match a prior `find()` call.

Instead, you should manually clear the cache after using those methods:

```php
$this->Model->deleteAll(array('Model.field' => true));
$this->Model->clearCache();
```

Or, iterate through ids and delete them individually (slower).

**Comment by jeremyharris** [comment]
Didn't mean to close this. Do you want to work on support for these methods? I think it may require a `find()` beforehand to determine ids, then clearing those specific ids. Likely not the most performant solution but I can't think of another one off the top of my head.
